waves/wallet/build.rs

```diff
@@ -41,16 +41,18 @@ fn main() -> Result<()> {
         &constants_rs,
         &format!(
             r#"
+use conquer_once::Lazy;
+
 pub const NATIVE_ASSET_TICKER: &str = "{}";
-pub const NATIVE_ASSET_ID: elements_fun::AssetId = elements_fun::AssetId::from_bytes({:?});
-pub const USDT_ASSET_ID: elements_fun::AssetId = elements_fun::AssetId::from_bytes({:?});
+pub static NATIVE_ASSET_ID: Lazy<elements::AssetId> = Lazy::new(|| elements::AssetId::from_slice(&{:?}).unwrap());
```

Comment:
Record an issue for this please.

Reply:
https://github.com/comit-network/waves/issues/135